src/configure/versionconfig.h.in

```diff
@@ -20,11 +20,13 @@ limitations under the License.
 
 namespace libcellml {
 
+// clang-format off
 #define LIBCELLML_VERSION_MAJOR @libCellML_VERSION_MAJOR@
 #define LIBCELLML_VERSION_MINOR @libCellML_VERSION_MINOR@
 #define LIBCELLML_VERSION_PATCH @libCellML_VERSION_PATCH@
 
-static unsigned int LIBCELLML_LIBRARY_VERSION=@LIBCELLML_LIBRARY_VERSION@;
-static const std::string LIBCELLML_LIBRARY_VERSION_STRING="@LIBCELLML_LIBRARY_VERSION_STRING@";
+static unsigned int LIBCELLML_LIBRARY_VERSION = @LIBCELLML_LIBRARY_VERSION@;
+// clang-format on
+static const std::string LIBCELLML_LIBRARY_VERSION_STRING = "@LIBCELLML_LIBRARY_VERSION_STRING@";
```

Comment:
What are the clang tidy failings here?

Reply:
Yes, sorry, I should have put a comment about this in my commit.

Basically, after ClangFormat has done its job,

```
static unsigned int LIBCELLML_LIBRARY_VERSION = @LIBCELLML_LIBRARY_VERSION@; 
```

will be

```
static unsigned int LIBCELLML_LIBRARY_VERSION = @LIBCELLML_LIBRARY_VERSION @;
```

i.e. there is a space before the second `@`, which in turn means that the file cannot be properly configured.